uploadfs.js

```diff
@@ -65,16 +64,45 @@ function Uploadfs() {
       self.cdn = options.cdn;
     }
 
+
     // Load image backend
-    self._image = options.image;
+     self._image = options.image;
+    const processors = [ 'imagemagick', 'jimp' ];
```

Comment:
This confused me. Call it `deprecatedProcessors`

Reply:
Removed in the new logic.